lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -1,9 +1,61 @@
 """
 Progress Tab Serializers
 """
-
 from rest_framework import serializers
 from lms.djangoapps.course_home_api.outline.v1.serializers import CourseBlockSerializer
+from rest_framework.reverse import reverse
+
+
+class ProblemScoresSerializer(serializers.Serializer):
+    earned = serializers.FloatField()
+    possible = serializers.FloatField()
+
+
+class GradedTotalSerializer(serializers.Serializer):
+    first_attempted = serializers.CharField()
+    possible = serializers.FloatField()
```

Comment:
Nit: If you don't mind, can you alphabetize these fields? I personally just find them easier to read that way because if you want to search, you know better where to look